### PR TITLE
Gh auth

### PR DESCRIPTION
**Pull request by OksentukVlad**

### CODE REVIEW

File: client/libs/github.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+// import { GitHub } from 'meteor/bruz:github-api';
+//
+// export default new GitHub({
+//   version: '3.0.0'
+// });
```
Comment (epaminond) [inline client/libs/github.js]: Do we need to keep this commented out code?

File: server/configs/service_configs.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import { ServiceConfiguration } from 'meteor/service-configuration';
+
+const githubConfig = {
+  clientId: '106f3d239e9292dd1281',
+  loginStyle: 'popup',
+  secret: 'd873a36f6514986656a5012a49721a9efc16df15'
```
Comment (epaminond) [inline server/configs/service_configs.js]: I guess we'd rather move clientId and secret out of the repo and pass them through Meteor.settings.

File: client/components/newpost/index.jsx [hunk 3]
```diff
@@ -3,7 +3,6 @@
 // To test this component, use React from npm directly
 // as shown below
```
Comment (epaminond) [inline client/components/newpost/index.jsx]: I think these comments are redundant.

File: server/configs/service_configs.js (hunk 2 above)
Comment (OksentukVlad) [inline server/configs/service_configs.js]: Yea, you're right. These are my clientId and secret. I use them for local development.

**Comment by yataras** [comment]
+1 Approve

**Comment by abaoaqus** [comment]
+1 Approve

**Pull request merged**